src/routes/transactions/entities/human-description.entity.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  InfoFragment,
+  ValueType,
+} from '@/domain/human-description/entities/human-description.entity';
+
+export interface RichInfoFragment extends InfoFragment {
+  richData: Record<string, unknown> | null;
+}
+
+export interface RichTokenValueFragment extends RichInfoFragment {
+  type: ValueType.TokenValue;
+  value: string;
+  richData: {
+    symbol: string | null;
+    logoUri: string | null;
+  };
```

Comment:
As we are not using extending from `RichInfoFragment` anymore I would also be in favour of flattening this since at the moment this is only being used by one fragment.